docs/changelog.md

```diff
@@ -14,6 +14,9 @@
 ### Breaking Changes
 * The `ContainerCreationConfigBuilder` constructor no longer takes in a test volume mountpoint
 * Added a `ContainerCreationConfigBuilder.WithTestVolumeMountpoint` for specifying the test volume mountpoint, which should be used instead if the default test volume mountpoint of `/kurtosis-test-volume` isn't acceptable
+* Replaced the argument `ContainerConfigFactory` in `AddService`and `AddServiceToPartition`with two arguments `ContainerCreationConfig`and an anonymous function which should returns `ContainerRunConfig`type;
+  users should use `ContainerCreationConfig` struct, and the function that was defined in `GetRunCongig` in the `ContainerConfigFactory` implementations as the new arguments
```

Comment:
Niiiiiice remediation steps! One note: this is on a newline, but that will confuse Markdown. Instead, I'd just put this as a sub-bulletpoint, e.g.:
```
* Replaced the argument `ContainerConfigFactory` in `AddService`and `AddServiceToPartition`with two arguments `ContainerCreationConfig`and an anonymous function which should returns `ContainerRunConfig`type
    * Users should use the `ContainerCreationConfig` struct, and the function that was defined in `GetRunConfig` in the `ContainerConfigFactory` implementations as the new arguments
```
(NOTE: the above also fixes a `GetRunCongig` typo)